.gitignore

```diff
@@ -52,3 +52,4 @@ lib/odc-*.jar
 maven/
 venv/
 **/local-unit-test.properties
+/.github/hooks/pre-push
```

Comment:
why should we add this?

Reply:
In Windows, I need to add a pre-push  to my github hooks, but others do not require this. Therefore, I have added it to .gitignore.